client/tailscale/localclient.go

```diff
@@ -1124,6 +1124,25 @@ func (lc *LocalClient) DeleteProfile(ctx context.Context, profile ipn.ProfileID)
 	return err
 }
 
+// EnableFeature makes a request for instructions on how to enable a
```

Comment:
Kinda a weird name in that it doesn't actually enable it.

It's more like "QueryFeatureAvailability" or something, which is kinda a mouthful. Maybe "QueryFeature" or "CheckFeature"?

Reply:
Yea I was going back and forth on naming too - I like `QueryFeature`, was also considering `RequestFeature` but then that's a lot of "request request" repetition

Reply:
Okay, updated to `QueryFeature`. I'm still not totally in love with the name, but can start here and update if I come up with a better one.